server/src/models/message.js

```diff
@@ -0,0 +1,27 @@
+import { DataTypes, UUIDV4 } from 'sequelize';
+import { client } from '../utils/db.js';
+import { text } from 'express';
```

Comment:
The import of `text` from 'express' is unnecessary and should be removed to avoid confusion, as it is not used in this file.